ci-operator/step-registry/openshift-qe/set-max-pods/openshift-qe-set-max-pods-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -euo pipefail
+set -o nounset
+set -o errexit
+set -o pipesfail
+
+MAX_PODS=500 # Set the desired maxPods value
+
+CLUSTER_NETWORK_CIDR=${CLUSTER_NETWORK_CIDR:-10.128.0.0/20}
+CLUSTER_NETWORK_HOST_PREFIX=${CLUSTER_NETWORK_HOST_PREFIX:-23}
+
+echo "default is to update cidr to 20 w/ hostPrefix of 23 so that max nodes num is 8"
+echo "\nsee below for actual values used\n--------------------------------\n"
+
+cat >> "${SHARED_DIR}/install-config.yaml" << EOF
```

Comment:
This step is supposed to config the max pods only. 
For the cidr and hostPrefix, it can not be done as post installation step. The install-config.yaml is used by the openshift installer. So it should be put in a step in the installation workflow. 
Please read the description in the Jira https://issues.redhat.com/browse/OCPQE-28897 '1. Config during cluster install' and let me know if you have some questions.